devices/tuya.js

```diff
@@ -2542,7 +2542,7 @@ module.exports = [
         toZigbee: [],
     },
     {
-        zigbeeModel: ['TS0004'],
+        fingerprint: [{modelID: 'TS0004', manufacturerName: '_TZ3000_ay8hciaf'}],
```

Comment:
This should make no difference, every TS0004 will already be detected as such.

Reply:
my apologies. I'm new to this